src/module_writer.rs

```diff
@@ -394,7 +394,7 @@ fn cffi_init_file() -> &'static str {
 import os
 from .ffi import ffi
 
-lib = ffi.dlopen(os.path.join(os.path.dirname(__file__), 'native.so'), 4098)
```

Comment:
Why was this `4098`? It raises `Invalid mode` on OpenBSD, it does not have `RTLD_NODELETE`.

Reply:
No idea tbh. I thought I copied that code from an example, but I can't find that anymore :eyes:.